bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -164,10 +146,14 @@ spec:
               - /bin/bash
               - -c
               - |
-                OVN_NODES_ARRAY=({{.OVN_NODES}})
-                if [[ "$K8S_NODE" == "${OVN_NODES_ARRAY[0]}" ]]; then
+                MASTER_IP="{{.OVN_MASTER_IP}}"
+                if [[ "${K8S_NODE_IP}" == "${MASTER_IP}" ]]; then
+                  USEIPV6=
```

Comment:
A comment from the old review was to use "0.0.0.0" here, which I guess would be a little more explicit about v4 vs v6